CONTRIBUTING.md

```diff
@@ -4,25 +4,21 @@
 
 We welcome additions and extensions to Doubtfire that help progress our goal of supporting student learning through frequent formative feedback and delayed summative assessment.
 
-This guide provides high-level details on how to contribute to the Doubtfire repositories.
+This guide provides high-level details on how to contribute to Thoth-Tech's Doubtfire repositories.
 
 ## Table of Contents
 
 - [Contributing to Doubtfire](#contributing-to-doubtfire)
   - [Table of Contents](#table-of-contents)
-  - [Getting started](#getting-started)
+  - [Doubtfire-deploy](#doubtfire-deploy)
```

Comment:
Great point. My reason for the change was to have `Getting Started` begin immediately with the instructions to get started - instead of the few paragraphs regarding the 3 repositories, and docker containers (since I felt these paragraphs better explain what doubtfire-deploy is, rather than getting started). Let me know your thoughts, and if there's a better solution for this.

Reply:
Thanks for explaining @PerryRose. I am fine either way 👍🏼